### DISCUSSION TITLE
vite搭建的microapp项目，提示Failed to resolve module specifier "/node_modules/vite/dist/client/env.mjs

### DISCUSSION THREAD

**Discussion by iamagentleboy** [jd-opensource/micro-app#836] [General]
microapp的url指定dev的url地址有问题，指定build的地址没问题

运行代码提示
提示 Uncaught TypeError: Failed to resolve module specifier "/node_modules/vite/dist/client/env.mjs". Invalid relative url or base scheme isn't hierarchical.

Uncaught TypeError: Failed to resolve module specifier "/node_modules/.vite/deps/vue.js?v=06f83340". Invalid relative url or base scheme isn't hierarchical.

**Comment by scottMan1001** [comment]
我遇到了相同问题。全程按照官方网配置
![image](https://github.com/micro-zoe/micro-app/assets/38151889/68f674d6-3438-4bb4-b50d-c25c9d1b75b8)
![image](https://github.com/micro-zoe/micro-app/assets/38151889/bd6e9f0f-e407-4e79-9abc-0775bfc2d7ae)

**Comment by yangfan-coder** [comment]
请问一下 解决了吗？我遇到相同的问题

**Comment by zhujiahuihui** [comment]
请问一下 解决了吗？我遇到相同的问题
